app/components/collections/show/header_component.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="d-flex justify-content-between">
+  <div>
+    <%= render Elements::HeadingComponent.new(level: :h1, text: title, classes: 'mb-4 d-inline-block') %>
+    <%= render Edit::EditIconButtonComponent.new(presenter:, classes: 'mx-2 py-0 mb-2', icon_classes: 'h4') %>
+
+    <span class="ms-4 fst-italic status"><%= status_message %></span>
+  </div>
+  <div class="d-flex align-items-start">
+    <%= render Edit::EditDraftButtonComponent.new(presenter:) %>
+    <%= render Edit::DiscardDraftButtonComponent.new(presenter:, classes: 'ms-2') %>
+    <% unless presenter.first_draft? %>
+      <% if helpers.allowed_to?(:create_work?, presenter.collection) %>
+        <%= render Elements::ButtonLinkComponent.new(link: new_work_path(collection_druid: presenter.druid), label: 'Deposit to this collection', variant: :primary, classes: 'ms-2') %>
```

Comment:
Could this just be `<%= render Elements::ButtonLinkComponent.new(...) if !first_draft? && allowed_to_create_work? %>`?